### PR TITLE
Added size prop to <InfoIcon/> and used it in <FormField/>

### PR DESCRIPTION
**Pull request by EgoziE**
Added feature:
https://github.com/wix/wix-style-react/issues/4083

### CODE REVIEW

File: src/FormField/FormField.visual.js [hunk 1]
```diff
@@ -17,6 +17,29 @@ const tests = [
       }),
     ],
   },
+  {
+    describe: 'Label sizes',
+    its: [
+      test('Label should me medium', {
```
Comment (nitayneeman) [inline src/FormField/FormField.visual.js]:
```suggestion
      test('Label should be medium', {
```

File: src/FormField/FormField.visual.js [hunk 2]
```diff
@@ -17,6 +17,29 @@ const tests = [
...
+        label: 'I am a medium label',
+        labelSize: 'medium',
+      }),
+      test('Label should me small', {
```
Comment (nitayneeman) [inline src/FormField/FormField.visual.js]:
```suggestion
      test('Label should be small', {
```

File: src/FormField/FormField.visual.js [hunk 3]
```diff
@@ -17,6 +17,29 @@ const tests = [
...
+        label: 'I am a small label',
+        labelSize: 'small',
+      }),
+      test('Label and info should me medium', {
```
Comment (nitayneeman) [inline src/FormField/FormField.visual.js]:
```suggestion
      test('Label and info should be medium', {
```

File: src/FormField/FormField.visual.js [hunk 4]
```diff
@@ -17,6 +17,29 @@ const tests = [
...
+        label: 'I am a medium label',
+        labelSize: 'medium',
+        infoContent: 'I am an info content',
+      }),
+      test('Label and info should me small', {
```
Comment (nitayneeman) [inline src/FormField/FormField.visual.js]:
```suggestion
      test('Label and info should be small', {
```

File: src/FormField/FormField.js [hunk 5]
```diff
@@ -126,13 +125,20 @@ class FormField extends React.Component {
             content: infoContent,
             ...infoTooltipProps,
           }}
+          iconSize={
+            {
+              [LABEL_SIZES.tiny]: '18px',
```
Comment (nitayneeman) [inline src/FormField/FormField.js]: Should be verified with UX

**Comment by mosheKerbel** [comment]
test3 & test4 are failing due to unrelated falky tests.

**Pull request merged**